api/v1/openapi.yaml

```diff
@@ -1394,6 +1394,10 @@ definitions:
         description: |
           The UUID for the expiration timer. Set when expiration has been
           enabled while allocating.
+      interface-number:
+        type: string
```

Comment:
Yeah, I went with string because it was too awkward to use anything else. For some reason, I wasn't even allowed to use an integer here and `make generate-api` failed with "you must use `string` or `array`".

Reply:
Swagger, you had one job... 

SGTM. Not ideal but I had a suspicion that this might be the answer. The only reason I thought there might be another way was that I saw some use of `type: integer` elsewhere in the openapi.yaml.

Reply:
Maybe it's because the other property fields are `string` or `array` :-/